src/SingleBookingPage/NearestBooking.js

```diff
@@ -27,21 +28,42 @@ const query = graphql`
 
 class NearestBooking extends React.Component<Props> {
   renderBooking = (renderState: RenderState) => {
-    if (renderState && renderState.error) {
-      return <BookingError />;
+    let content = null;
+    const booking = idx(renderState.props, _ => _.nearestBooking);
+
+    if (booking) {
+      content = <BookingDetail booking={booking} />;
+    } else if (!booking) {
+      content = <BookingNotFound />;
     }
 
     if (!renderState.props) {
-      return <Loader />;
+      content = (
+        <ContentLoader
```

Comment:
Code looks totally good too!

Just one idea: maybe we should create a custom component that is based on `<ContentLoader>`, but doesn't require us to specify `speed` and the colors. It's not a priority now as it's only used in one place, but once we have multiple "skeletons" it might be annoying. The same we have/had with media queries copy-pasted to every file with the same values.

Reply:
I would move it to separate component as well, to keep this file smaller, not issue for now